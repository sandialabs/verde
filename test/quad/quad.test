#TEST_KEY QUAD ALL
quad1.gen
quad2.gen
quad3.gen
quad4.gen
