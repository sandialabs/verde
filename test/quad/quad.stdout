


		         *** VERDE Version 2.6 ***

		        Revised 06-17-02 10:50:45 AM

		VERDE: VERIFICATION OF DISCRETE ELEMENTS

		A MESH VERIFICATION TOOL FOR EXODUS MESHES

		      Executing on 06/17/02 at 10:54:31 AM

		This version of Verde is only for use inside Sandia

Executing: ../../verde_exe -batch -nodefaults quad1.gen 

Processing File: quad1.gen...
Reading block: 1,
Processing Skin...

---------------
Element Metrics
---------------
Processing block 1, 195 elements of type SHELL4:


Summary Quad Metrics, total = 195 Quads:

   Function Name     Average    Std Dev    Minimum   (id)  Maximum  (id)
 -----------------  ---------  ---------  --------------  -------------

ALGEBRAIC METRICS
             Shear  0.9601829   0.06175   0.6202990 (9)  0.9998407 (15)
             Shape  0.9435260   0.07368   0.6096954 (9)  0.9972970 (18)
     Relative Size  0.7015127   0.19926   0.1838892 (155)  0.9981684 (105)
    Shape and Size  0.6680510   0.20874   0.1657578 (155)  0.9946882 (105)

DIAGNOSTIC METRICS
      Element Area  0.0056914   0.00149   0.0024406 (155)  0.0120975 (153)

TRADITIONAL METRICS
      Aspect Ratio  1.1512230   0.14525   1.0007210 (175)  1.7371291 (7)
              Skew  0.1008179   0.10740   0.0005134 (37)  0.5289187 (195)
             Taper  0.0704745   0.06397   0.0026003 (172)  0.3283511 (169)
           Warpage  1.0000000   0.00000   1.0000000 (1)  1.0000000 (1)
           Stretch  0.8582606   0.08741   0.6067221 (190)  0.9628850 (104)
     Maximum Angle  101.30988   10.4937   90.609311 (15)  141.66203 (9)
     Minimum Angle  79.780596   8.45266   50.869762 (190)  89.331562 (33)
              Oddy  0.2994284   0.48099   0.0108559 (18)  3.3802706 (9)
     Condition No.  1.0675711   0.10029   1.0027103 (18)  1.6401632 (9)
          Jacobian  0.0050916   0.00143   0.0016948 (177)  0.0102650 (153)
   Scaled Jacobian  0.9601829   0.06175   0.6202990 (9)  0.9998407 (15)

---------------
Metrics Summary
---------------
   Number of Failed Quad Metrics: 40
      Failed Relative Size: 33  	(valid range:      0.500 to      1.000)
      Failed Shape and Size: 4  	(valid range:      0.200 to      1.000)
      Failed Skew: 2  	(valid range:      0.000 to      0.500)
      Failed Maximum Angle: 1  	(valid range:     90.000 to    135.000)

----------------------------
Quads Sharing 3 Nodes
----------------------------
   Number Quads sharing three nodes:  0

----------------------------
Mesh Topology Information
----------------------------
   Probable Topology:  Open Surface with 1 hole(s)
   Possible Topologies: N Open surfaces with N hole(s)

-------------------------
Mesh Interface Information
-------------------------
   Number Surface Quads:     195
   Number Surface Tris:      0
   Number Coincident Nodes:  0
   Number Coincident Quads:   0
   Number Coincident Tris:   0
   Number Model Edges:       72



-----------------
Execution Summary
-----------------
   User time             = 0.01
   Elements Processed    = 195
   Elements Per Second   = 19500
   Total Area            = 1.11

   Metrics Tests         = FAIL  (see Metrics Summary above)
   Mesh Topology Tests   = PASS
   Mesh Interface Tests  = PASS
   VERDE                 = FAIL



		         *** VERDE Version 2.6 ***

		        Revised 06-17-02 10:50:45 AM

		VERDE: VERIFICATION OF DISCRETE ELEMENTS

		A MESH VERIFICATION TOOL FOR EXODUS MESHES

		      Executing on 06/17/02 at 10:54:31 AM

		This version of Verde is only for use inside Sandia

Executing: ../../verde_exe -batch -nodefaults quad2.gen 

Processing File: quad2.gen...
Reading block: 1,
Processing Skin...

---------------
Element Metrics
---------------
Processing block 1, 515 elements of type SHELL4:


Summary Quad Metrics, total = 515 Quads:

   Function Name     Average    Std Dev    Minimum   (id)  Maximum  (id)
 -----------------  ---------  ---------  --------------  -------------

ALGEBRAIC METRICS
             Shear  0.9503628   0.05575   0.7001339 (244)  0.9997747 (91)
             Shape  0.9387705   0.06415   0.6150244 (350)  0.9996665 (476)
     Relative Size  0.7145315   0.21564   0.1494347 (496)  0.9987997 (80)
    Shape and Size  0.6772855   0.22182   0.1290211 (368)  0.9970367 (476)

DIAGNOSTIC METRICS
      Element Area  25.280606   6.82740   9.7726706 (496)  56.299628 (368)

TRADITIONAL METRICS
      Aspect Ratio  1.1195168   0.11354   1.0002929 (476)  2.1295568 (350)
              Skew  0.1490940   0.10146   0.0003769 (6)  0.5749651 (319)
             Taper  0.0760016   0.06799   0.0007944 (184)  0.3142901 (13)
           Warpage  1.0000000   0.00000   1.0000000 (1)  1.0000000 (1)
           Stretch  0.8466153   0.07248   0.6012965 (41)  0.9863180 (476)
     Maximum Angle  104.66620   9.08194   90.888417 (274)  135.56225 (244)
     Minimum Angle  76.720364   7.28900   48.088378 (319)  89.535815 (91)
              Oddy  0.3086308   0.40243   0.0013346 (476)  3.2874378 (350)
     Condition No.  1.0710002   0.08537   1.0003336 (476)  1.6259517 (350)
          Jacobian  22.359496   6.25246   6.6328157 (495)  41.984018 (372)
   Scaled Jacobian  0.9503628   0.05575   0.7001339 (244)  0.9997747 (91)

---------------
Metrics Summary
---------------
   Number of Failed Quad Metrics: 104
      Failed Relative Size: 91  	(valid range:      0.500 to      1.000)
      Failed Shape and Size: 11  	(valid range:      0.200 to      1.000)
      Failed Skew: 1  	(valid range:      0.000 to      0.500)
      Failed Maximum Angle: 1  	(valid range:     90.000 to    135.000)

----------------------------
Quads Sharing 3 Nodes
----------------------------
   Number Quads sharing three nodes:  0

----------------------------
Mesh Topology Information
----------------------------
   Probable Topology:  Open Surface with 1 hole(s)
   Possible Topologies: N Open surfaces with N hole(s)

-------------------------
Mesh Interface Information
-------------------------
   Number Surface Quads:     515
   Number Surface Tris:      0
   Number Coincident Nodes:  0
   Number Coincident Quads:   0
   Number Coincident Tris:   0
   Number Model Edges:       212



-----------------
Execution Summary
-----------------
   User time             = 0.03
   Elements Processed    = 515
   Elements Per Second   = 17166
   Total Area            = 13019.51

   Metrics Tests         = FAIL  (see Metrics Summary above)
   Mesh Topology Tests   = PASS
   Mesh Interface Tests  = PASS
   VERDE                 = FAIL



		         *** VERDE Version 2.6 ***

		        Revised 06-17-02 10:50:45 AM

		VERDE: VERIFICATION OF DISCRETE ELEMENTS

		A MESH VERIFICATION TOOL FOR EXODUS MESHES

		      Executing on 06/17/02 at 10:54:31 AM

		This version of Verde is only for use inside Sandia

Executing: ../../verde_exe -batch -nodefaults quad3.gen 

Processing File: quad3.gen...
Reading block: 1,
Processing Skin...

---------------
Element Metrics
---------------
Processing block 1, 1877 elements of type SHELL4:


Summary Quad Metrics, total = 1877 Quads:

   Function Name     Average    Std Dev    Minimum   (id)  Maximum  (id)
 -----------------  ---------  ---------  --------------  -------------

ALGEBRAIC METRICS
             Shear  0.9416248   0.09007   0.2675522 (1284)  1.0000000 (95)
             Shape  0.9275001   0.10506   0.2674975 (1284)  0.9999639 (1539)
     Relative Size  0.6188160   0.23978   0.0053350 (889)  0.9974514 (474)
    Shape and Size  0.5816998   0.24428   0.0048138 (889)  0.9936554 (1672)

DIAGNOSTIC METRICS
      Element Area  31.250548   10.5646   2.2825147 (889)  70.688566 (793)

TRADITIONAL METRICS
      Aspect Ratio  1.1250073   0.18132   1.0000409 (1212)  2.9955143 (1333)
              Skew  0.1273600   0.14797   0.0000142 (95)  0.7631947 (1314)
             Taper  0.0806885   0.08799   0.0000006 (106)  0.4457981 (91)
           Warpage  0.9931349   0.06147   0.0998256 (588)  1.0000000 (121)
           Stretch  0.8536078   0.11730   0.3569872 (627)  0.9965627 (727)
     Maximum Angle  103.29910   13.0407   90.001366 (106)  164.46406 (1284)
     Minimum Angle  77.948356   11.4770   33.473439 (1774)  89.998688 (106)
              Oddy  0.4636021   1.05641   0.0001443 (1539)  24.382092 (1284)
     Condition No.  1.0977079   0.18000   1.0000361 (1539)  3.7383526 (1284)
          Jacobian  28.010984   10.9947   1.5727830 (892)  68.589487 (793)
   Scaled Jacobian  0.9413892   0.09057   0.2674043 (1284)  1.0000000 (96)

---------------
Metrics Summary
---------------
   Number of Failed Quad Metrics: 877
      Failed Shear: 1  	(valid range:      0.300 to      1.000)
      Failed Shape: 1  	(valid range:      0.300 to      1.000)
      Failed Relative Size: 497  	(valid range:      0.500 to      1.000)
      Failed Shape and Size: 191  	(valid range:      0.200 to      1.000)
      Failed Skew: 81  	(valid range:      0.000 to      0.500)
      Failed Warpage: 22  	(valid range:      0.900 to      1.000)
      Failed Maximum Angle: 49  	(valid range:     90.000 to    135.000)
      Failed Minimum Angle: 33  	(valid range:     45.000 to     90.000)
      Failed Oddy: 1  	(valid range:      0.000 to     16.000)
      Failed Scaled Jacobian: 1  	(valid range:      0.500 to      1.000)

----------------------------
Quads Sharing 3 Nodes
----------------------------
   Number Quads sharing three nodes:  0

----------------------------
Mesh Topology Information
----------------------------
   Probable Topology:  Open Surface with 11 hole(s)
   Possible Topologies: N Open surfaces with N+10 hole(s)

-------------------------
Mesh Interface Information
-------------------------
   Number Surface Quads:     1877
   Number Surface Tris:      0
   Number Coincident Nodes:  0
   Number Coincident Quads:   0
   Number Coincident Tris:   0
   Number Model Edges:       584



-----------------
Execution Summary
-----------------
   User time             = 0.11
   Elements Processed    = 1877
   Elements Per Second   = 17063
   Total Area            = 58657.28

   Metrics Tests         = FAIL  (see Metrics Summary above)
   Mesh Topology Tests   = PASS
   Mesh Interface Tests  = PASS
   VERDE                 = FAIL



		         *** VERDE Version 2.6 ***

		        Revised 06-17-02 10:50:45 AM

		VERDE: VERIFICATION OF DISCRETE ELEMENTS

		A MESH VERIFICATION TOOL FOR EXODUS MESHES

		      Executing on 06/17/02 at 10:54:31 AM

		This version of Verde is only for use inside Sandia

Executing: ../../verde_exe -batch -nodefaults quad4.gen 

Processing File: quad4.gen...
Reading block: 1,
Processing Skin...

---------------
Element Metrics
---------------
Processing block 1, 100 elements of type SHELL4:


Summary Quad Metrics, total = 100 Quads:

   Function Name     Average    Std Dev    Minimum   (id)  Maximum  (id)
 -----------------  ---------  ---------  --------------  -------------

ALGEBRAIC METRICS
             Shear  0.9848611   0.01996   0.9245308 (10)  0.9999999 (18)
             Shape  0.9760813   0.01976   0.9223196 (10)  0.9997119 (23)
     Relative Size  0.8722310   0.08774   0.6700101 (8)  0.9984886 (38)
    Shape and Size  0.8511855   0.08610   0.6636525 (8)  0.9968688 (38)

DIAGNOSTIC METRICS
      Element Area  0.1888479   0.01721   0.1627271 (56)  0.2307127 (8)

TRADITIONAL METRICS
      Aspect Ratio  1.1253275   0.06957   1.0125292 (82)  1.2722917 (51)
              Skew  0.0931350   0.09990   0.0000000 (3)  0.3345578 (96)
             Taper  0.0188901   0.01066   0.0003657 (18)  0.0346255 (51)
           Warpage  0.9691343   0.02073   0.9309374 (21)  1.0000000 (3)
           Stretch  0.8965024   0.05092   0.8100101 (6)  0.9864437 (23)
     Maximum Angle  96.048159   5.73264   90.018827 (18)  109.92016 (10)
     Minimum Angle  83.097697   6.43616   67.797278 (6)  89.981173 (18)
              Oddy  0.0949660   0.08765   0.0011530 (23)  0.3460687 (6)
     Condition No.  1.0249291   0.02118   1.0002882 (23)  1.0842228 (10)
          Jacobian  0.1838676   0.01832   0.1567104 (50)  0.2302307 (8)
   Scaled Jacobian  0.9830973   0.02004   0.9226668 (6)  0.9999999 (18)

---------------
Metrics Summary
---------------
   Number of Failed Quad Metrics: 0

----------------------------
Quads Sharing 3 Nodes
----------------------------
   Number Quads sharing three nodes:  0

----------------------------
Mesh Topology Information
----------------------------
   Probable Topology:  1 Open Surface(s)
   Possible Topologies: N Open surfaces with N-1 hole(s)

-------------------------
Mesh Interface Information
-------------------------
   Number Surface Quads:     100
   Number Surface Tris:      0
   Number Coincident Nodes:  0
   Number Coincident Quads:   0
   Number Coincident Tris:   0
   Number Model Edges:       88



-----------------
Execution Summary
-----------------
   User time             = 0.01
   Elements Processed    = 100
   Elements Per Second   = 10000
   Total Area            = 18.88

   Metrics Tests         = PASS
   Mesh Topology Tests   = PASS
   Mesh Interface Tests  = PASS
   VERDE                 = PASS
